File: src/main/java/com/qrware/dto/ZoneDTO.java
package com.qrware.dto;

import com.qrware.domain.warehouse.ZoneType;
import java.time.LocalDateTime; // Upewnij się, że to jest LocalDateTime

public record ZoneDTO(
        Long id,
        String name,
        String code,
        String description,
        ZoneType type,
        Boolean active,
        Boolean temperatureControlled,
        Integer temperatureMin,
        Integer temperatureMax,
        Boolean humidityControlled,
        Integer humidityMin,
        Integer humidityMax,
        Integer securityLevel,
        Boolean hazardousMaterials,
        Boolean fragileItems,
        Integer pickingPriority,
        String manager,
        String contactInfo,
        String color,
        LocalDateTime createdAt, // Sprawdź ten typ
        LocalDateTime updatedAt, // Sprawdź ten typ
        String createdBy,
        String updatedBy,
        int locationCount,
        long activeLocationCount,
        long occupiedLocationCount,
        double occupancyRate
) {
}

File: Wareapp/app/src/main/java/com/qrware/app/data/dto/ZoneDTO.kt
package com.qrware.app.data.dto

import com.qrware.app.data.model.ZoneType
import java.time.LocalDateTime

data class ZoneDTO(
    val id: Long?,
    val name: String,
    val code: String,
    val description: String?,
    val type: ZoneType,
    val active: Boolean,
    val temperatureControlled: Boolean,
    val temperatureMin: Int?,
    val temperatureMax: Int?,
    val humidityControlled: Boolean,
    val humidityMin: Int?,
    val humidityMax: Int?,
    val securityLevel: Int,
    val hazardousMaterials: Boolean,
    val fragileItems: Boolean,
    val pickingPriority: Int,
    val manager: String?,
    val contactInfo: String?,
    val color: String?,

    val createdAt: String?,
    val updatedAt: String?,
    val createdBy: String?,

    val locationCount: Int,
    val activeLocationCount: Long,
    val occupiedLocationCount: Long,
    val occupancyRate: Double
)